src/ml_board/backend/dummy_server/dummy_server.py

```diff
@@ -0,0 +1,162 @@
+####################################################################################################
+# imports
+import json
+import sys
+from collections import defaultdict
+from typing import List
+
+from flask import Flask, copy_current_request_context, render_template, request
+from flask_socketio import SocketIO, disconnect, emit, join_room, rooms
+
+####################################################################################################
+# TODO: ASK Max what's that for?
+# class EventSubscriberIF:
+#     def callback(self):
+#         raise NotImplementedError
+
+
+####################################################################################################
+# Flask HTTP Server
+def CreateFlaskServer() -> Flask:
+    # create Flask server
+    _app: Flask = Flask(__name__, template_folder="template")
+    _app.secret_key = 'secret!'
+
+    # set routes with their respective callbacks
+    _app.add_url_rule('/', view_func=_index)
+    # _app.add_url_rule('/status', view_func=_status)
+    # _app.add_url_rule('/api/status', view_func=_api_status)
+
+    return _app
+
+
+def _index() -> str:
+    # return render_template('index.html', async_mode=ws._socketio.async_mode)
+    return "Index Page"
+
+
+# def _status() -> str:
```

Comment:
lots of legacy code that needs to be removed before merging

Reply:
I'll delete **_status** but **_api_status** was in the old code but not used and I'm not sure how was it intended to be used?!!?! 🤷